src/builders/residfp-builder/residfp/EnvelopeGenerator.h

```diff
@@ -169,7 +169,8 @@ class EnvelopeGenerator
         decay(0),
         sustain(0),
         release(0),
-        env3(0)
+        env3(0),
+        dac()
```

Comment:
dac is initialized when setChipModel is called from the SID constructor so there's no chance of uninitialized accesses, like it's done in WaveformGenerator (not sure why it's not flagged there).
Maybe it would be better to mark the variable with //-V730_NOINIT

Reply:
This is just default initialization of array, it should not change behavior, just suppress any warnings or style linters.
Correct me, if i'm wrong.

Reply:
PVS is a static inspector, and gives recommendations on code quality regardless of how the overall program hangs together.

In this case, a user can construct an `Envelope e` object, then call `e.output()` and get handed and uninitialzed `dac[0xaa]` float.

Reply:
> This is just default initialization of array, it should not change behavior, just suppress any warnings or style linters.
> Correct me, if i'm wrong.

You're right, it would initialize the array with zeros, but it will be then overwritten when setChipModel is called so it's just wasting time.


> However taken on its own, a user can construct an `Envelope` object and immediately call its `output()` member function, and get handed and uninitialized `dac[0xaa]` float.

Sure but this is not the case here. Anyway as mentioned in issue #34 I plan to rework this part so it doesn't really matter.

Reply:
> I plan to rework this part so it doesn't really matter.
:+1:

Reply:
Please revert this one as it will be solved in another way as explained above.